test/functional/commands/keyboard/keyboard-e2e-specs.js

```diff
@@ -49,6 +60,55 @@ async function runTextEditTest (driver, testText, keys = false) {
   return el;
 }
 
+/*
+ * The key event page needs to be cleared between runs, or else we get false
+ * positives from previously run tests. The page has a single button that
+ * removes all text from within the main TextView.
+ */
+async function clearKeyEvents (driver) {
+  let el = await getElement(driver, BUTTON_CLASS);
+  driver.click(el);
+
+  // wait a moment for the clearing to occur, lest we too quickly try to enter more text
+  await B.delay(500);
+}
+
+async function runCombinationKeyEventTest (driver) {
+  let runTest = async function () {
```

Comment:
Ah. There is no gain from that, here. The function isn't relying on any scoping.